CODEOWNERS

```diff
@@ -67,3 +67,6 @@
 
 # docs
 /man/openmp-utils.Rd @Anirban166
+
+# meta
+/.graphics/translation-team-logos/*.png @tdhock
```

Comment:
@jmrung could you please volunteer as reviewer under .graphics, since you created the new logos?